### DISCUSSION TITLE
Pack multiple calldatas

### DISCUSSION THREAD

**Discussion by zeroknots** [PaulRBerg/prb-proxy#52] [Q&A] [answered]
Trying to figure out how I can call execute() with multiple function calls to the target contract.

All the tests in the repo, are only using one `interface.encodeFunctionData`

how can pack multiple calldatas together to create multiple transactions in one atomic tx

**Comment by PaulRBerg** [comment, marked as answer]
Hello! You can only call one function at a time with `execute`, but the function that you are calling can bundle up multiple function calls.

Refer to the [Targets](https://github.com/PaulRBerg/prb-proxy/wiki/targets) wiki for guidance on how to write target contracts.

**Reply by cygnusv** [reply]
The Targets wiki only shows a Lorem ipsum 😅

**Reply by PaulRBerg** [reply]
It will be updated soon when I ship PRBProxy V4.
